### PR TITLE
Added a replay and more info button.

### PR DESCRIPTION
**Pull request by Hugovs2000**
On the page where you can select a lap, I added two buttons. The replay button reruns the animation of the go-kart going around the track. When in mobile view, a button displays at the top of the map and when clicked a dropdown shows more lap details that is normally displayed in desktop view.

### CODE REVIEW

File: src/dom/ui-manip.ts [hunk 1]
```diff
@@ -210,3 +209,96 @@ export function toggleActiveButton(id: string) {
     }
   }
 }
+const mapContainer = document.getElementById("map-container");
+
+export function createResetButton() {
+  const resetButton = document.createElement("button");
+
+  resetButton.id = "resetButton";
```
Comment (tanadendere) [inline src/dom/ui-manip.ts]: CSS naming convention: kebab case🍢

File: src/dom/ui-manip.ts [hunk 2]
```diff
@@ -210,3 +209,96 @@ export function toggleActiveButton(id: string) {
...
+    const checkResetButton = document.getElementById("resetButton");
+    if (checkResetButton) {
+      mapContainer.removeChild(checkResetButton);
+    }
+  }
+}
+
+export function moreInfoDropdown(runsJSON: IKartLapsPerRun, lapNum: number) {
+  const moreInfo = document.createElement("div");
+  moreInfo.id = "moreInfo";
```
Comment (tanadendere) [inline src/dom/ui-manip.ts]: Kebab case here as well

**Review by tanadendere — APPROVED** [review]
Looks good! Just the naming convections .

File: src/dom/ui-manip.ts (hunk 1 above)
Comment (Hugovs2000) [reply src/dom/ui-manip.ts]: Good spot 👍🏻

**Review by yanevdw — APPROVED** [review]
Other than the naming conventions mentioned, really well done.

**Pull request merged**